packages/ui-navigation/src/Navigation/NavigationItem/props.ts

```diff
@@ -32,6 +32,7 @@ import type {
 import type { WithStyleProps, ComponentStyle } from '@instructure/emotion'
 
 type NavigationItemOwnProps = {
+  elementRef?: (el: Element | null) => void
```

Comment:
Please add new props to the `allowedProps` array too

Reply:
done